Frontend/library/src/AFK/AFKController.test.ts

```diff
@@ -34,66 +36,71 @@ describe('AFKController', () => {
         expect(mockPixelStreaming.dispatchEvent).not.toHaveBeenCalled();
     });
 
-    it('should activate AFK timer and trigger it after specified delay if it has been enabled from settings', () => {
-        const timeoutSeconds = 100;
-
-        const config = new Config({ initialSettings: { [Flags.AFKDetection]: true, [NumericParameters.AFKTimeoutSecs]: timeoutSeconds} });
-        const onDismissAfk = jest.fn();
-        const afkController = new AFKController(config, mockPixelStreaming, onDismissAfk);
-
-        afkController.startAfkWarningTimer();
-        expect(afkController.active).toBe(true);
-
-        // Advance to 1 second before AFK event:
-        jest.advanceTimersByTime((timeoutSeconds - 1) * 1000);
-        expect(mockPixelStreaming.dispatchEvent).not.toHaveBeenCalled();
-        
-        // advance 1 more second to trigger AFK warning
-        jest.advanceTimersByTime(1000);
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningActivateEvent({
-            countDown: 0,
-            dismissAfk: expect.anything()
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 10,
-        }));
-
-        // advance 10 more seconds to trigger AFK countdown updates and eventually timeout
-        jest.advanceTimersByTime(10 * 1000);
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 9,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 8,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 7,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 6,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 5,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 4,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 3,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 2,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkWarningUpdateEvent({
-            countDown: 1,
-        }));
-        expect(mockPixelStreaming.dispatchEvent).toHaveBeenCalledWith(new AfkTimedOutEvent());
-    });
+    it('should activate AFK timer and trigger it after specified delay if it has been enabled from settings',
+       () => {
```

Comment:
Not a huge fan of forcing the lambda onto the next line here.

Reply:
Not sure if its obvious because GH does word wrap itself, but its on a new line because it doesn't break long strings like this description here, and because it's beyond the max line length it breaks when it next can.